src/app/contextBroker/contextBroker.cpp

```diff
@@ -373,6 +374,10 @@ PaArgument paArgs[] =
 #define SSR_COMPS_V2            2, { "v2", "subscriptions" }
 #define SSR_COMPS_WORD          ""
 
+#define ISR                     SubscriptionRequest
+#define ISR_COMPS_V2            3, { "v2", "subscriptions", "*" }
+#define ISR_COMPS_WORD          ""
+
```

Comment:
I imagine the 'I' in 'ISR' stands for *Individual*.
If so, IndividualSubscriptionRequest would be a better name for the "ISR constant".